pkg/logging/klog/logger.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package klog implements logr using klog.
+// This particular implementation is mostly copy pasta'd from https://github.com/go-logr/glogr/blob/master/glogr.go
```

Comment:
making sure this line is seen

Reply:
Any reason for the custom implementation here vs using [klogr](https://github.com/kubernetes/klog/blob/master/examples/klogr/main.go)?

Reply:
oops, no, failed to notice that dir in `klog`. However, I do prefer dealing with structs instead of things like `New` that return interfaces, but I can live with it.

Updating PR, thanks for pointing that out!